src/test/ui/track-diagnostics/track2.stderr

```diff
@@ -7,6 +7,11 @@ LL |     let _moved @ _from = String::from("foo");
    |         |        value moved here
    |         value used here after move
 -Ztrack-diagnostics: created at compiler/rustc_borrowck/src/borrowck_errors.rs:LL:CC
+   |
+help: borrow this binding in the pattern to avoid moving the value
+   |
+LL |     let _moved @ ref _from = String::from("foo");
```

Comment:
This is an erroneous suggestion -- perhaps we can detect the case of `@ RHS` and not suggest `ref RHS` if so?